babel.config.js

```diff
@@ -35,6 +35,7 @@ module.exports = function (api /*: ApiType */) /*: BabelConfig */ {
         '@adeira/babel-preset-adeira',
         {
           target: api.caller(isWebpack) ? 'js-esm' : 'js',
+          reactRuntime: 'automatic',
```

Comment:
We should probably swich it to `automatic` by default after this PR. Followup issue? :)

Reply:
#1253